webapp/src/modules/parcels/utils.js

```diff
@@ -1,21 +1,14 @@
-const emptyParcel = {
-  x: null,
-  y: null,
-  price: null,
-  owner: null,
-  district_id: null,
-  data: {
-    name: '',
-    description: '',
-    ipns: null,
-    version: 0
+export function cleanParcel(parcel) {
```

Comment:
I don't get the change here! `cleanParcel` isn't fulfilling the same function as `emptyParcel` did

Reply:
`emptyParcel` was used when not all the endpoints returned the parcels with the same stuff (ie, some parcels might come with `data` or `publication`) so we merged them with the parcels from the previous state, and `emptyParcel` was like all the "default" values. That's not needed anymore since all the endpoints return the parcels the same way, so we can just replace the old parcels with the new ones instead of "merging" them. 

What `cleanParcel` is doing basically is removing the `publication` and replacing it with `publication_tx_hash` (because publications are not stored inside the parcel, but in their own part of the state).